easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.14-goolf-1.4.10-Perl-5.16.3.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'ConfigureMake'
+
+name = 'VCFtools'
+version = '0.1.14'
+
+homepage = 'https://vcftools.github.io/'
+description = """A set of tools written in Perl and C++ for working with VCF files."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://github.com/vcftools/vcftools/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+
+perl = 'Perl'
+perlver = '5.16.3'
+versionsuffix = '-%s-%s' % (perl, perlver)
+
+dependencies = [
+    (perl , perlver),
+]   
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["vcftools", "fill-fs", "vcf-compare", "vcf-contrast", 
+                                     "vcf-fix-ploidy", "vcf-merge"]],
+    'dirs': [],
+}
+
+modextrapaths = { 'PERL5LIB': 'lib/perl5/site_perl/%s/' % (perlver) }
```

Comment:
existing easyconfig for VCFtools 0.1.14 has this instead:

```python
modextrapaths = {'PERL5LIB': 'lib/perl5/site_perl'}
```

One of these is wrong?

Also, style-wise this should be:

```python
modextrapaths = {'PERL5LIB': 'lib/perl5/site_perl/%s' % perlver}
```